README.md

```diff
@@ -224,6 +224,39 @@ If your repo has a different command to run for migrations, you can configure it
 `DeclareSchema.db_migrate_command = "bundle exec rails db:migrate_immediate"`
 ```
 
+## The `belongs_to` Relation and Indexes
+By default, `declare_schema` creates an index for `belongs_to` relations. If an index
+subsequently needs to be removed, it must be specifically updated in the relation statement.
```

Comment:
I wouldn't focus on removing or adding. I'd just say "If this index is not desired, set `index: false`." We could cite the most common example for that, too, which is when a different index already covers this column at the front, as in:
```ruby
index [:other_table, :name]
```
Also, BTW, let's not use `belongs_to ... null: false` in the example, since Rails now has their own notation for this: `optional: false`. (`declare_schema` supports both, but just for backward-compatibility.)

Reply:
Section rewritten